packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -2,7 +2,7 @@
 
 const path = require('path');
 
-module.exports = {
+module.exports = (themeOptions = {}) => ({
```

Comment:
It like the pattern to define all allowed options with defaults, so it's much safer to track whether the documentation of the options is correct and matches the implementation. 

`themeOptions = { gaTrackingId: false }`  ?

Reply:
Sure, makes sense

Reply:
I added this here: https://github.com/commercetools/commercetools-docs-kit/pull/62/files#diff-829e90677cff64de530aac85e640cabcR20-R21